utils/update_translation_stats.py

```diff
@@ -85,68 +82,63 @@ def generate_translation_stats(po_dir, output_file):
                 break
             header = header + ' ' + line
```

Comment:
A question: does pocount really "distributes its header over multiple lines"? I have never seen this. Not in its bug tracker, not in any output on my machine, not in any example online. Can we get rid of this? In case it really happens, the code will now (since 62addf1487373448defea9e5cfa7d13b6b762795) detect the missing columns and report it.

Reply:
Since we tell it to use the CSV format, it would actually be a major bug if it did… Maybe the terminal line wrapping confused the original author of this?

IMO you can try removing it.